.github/workflows/integration-tests.yml

```
@@ -124,11 +124,17 @@ jobs:
         with:
           name: coverage-artifacts-html
           path: '/var/tmp/report'
-      
+
       - name: Upload coverage file
         if: always()
         uses: actions/upload-artifact@v3
         with:
           name: coverage-artifacts-info
           path: '/var/tmp/merged.info'
-      
\ No newline at end of file
+
+      - name: Report to Coveralls
+        if: always()
+        uses: coverallsapp/github-action@v2
+        with:
+          file: '/var/tmp/merged.info'
+          format: lcov
```

According to [coveralls-github-action](https://github.com/marketplace/actions/coveralls-github-action), the github token is required:
```suggestion
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          file: '/var/tmp/merged.info'
          format: lcov
```

---

It's strange. On one hand they say it's required. But, on the other, it has a default value. Anyhow, I've accepted the change.